docs/source/DeveloperDocumentation/PackagingGuides/shaka-packager.md

```diff
@@ -1 +1,326 @@
+
+### Examples of Each Document
+
+#### CPIX Document
+
+Each DRM Provider has a unique `systemId`. This helps to distinguish which values belong to which DRM Provider in a CPIX Document. 
+
+```
+System IDs:
+Widevine: edef8ba9-79d6-4ace-a3c8-27dcd51d21ed
+PlayReady: 9a04f079-9840-4286-ab92-e65be0885f95
+FairPlay: 94ce86fb-07ff-4f43-adb8-93d2fa968ca2
+```
+
+Below is an example of a CPIX Document with all `DRM Providers` selected. 
+
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<cpix:CPIX xmlns:pskc="urn:ietf:params:xml:ns:keyprov:pskc" xmlns:xsi="urn:ietf:params:xml:ns:keyprov:pskc" xmlns:cpix="urn:dashif:org:cpix" xsi:schemaLocation="urn:dashif:org:cpix cpix.xsd">
+    <cpix:ContentKeyList>
+        <cpix:ContentKey kid="3e4a4bfb-a252-a4b4-c839-6c8954d986f5" explicitIV="MjlkMzNlYzVjYWIyZmRmZg==" commonEncryptionScheme="cenc">
+            <cpix:Data>
+                <pskc:Secret>
+                    <pskc:PlainValue>3wFFjPspsydY/t2uSPeE6w==</pskc:PlainValue>
+                </pskc:Secret>
+            </cpix:Data>
+        </cpix:ContentKey>
+    </cpix:ContentKeyList>
+    <cpix:DRMSystemList>
+        <cpix:DRMSystem kid="3e4a4bfb-a252-a4b4-c839-6c8954d986f5" systemId="9a04f079-9840-4286-ab92-e65be0885f95">
+            <cpix:PSSH><PlayReady_PSSH></cpix:PSSH>
+        </cpix:DRMSystem>
+        <cpix:DRMSystem kid="3e4a4bfb-a252-a4b4-c839-6c8954d986f5" systemId="edef8ba9-79d6-4ace-a3c8-27dcd51d21ed">
+            <cpix:PSSH><Widevine_PSSH></cpix:PSSH>
+        </cpix:DRMSystem>
+        <cpix:DRMSystem kid="3e4a4bfb-a252-a4b4-c839-6c8954d986f5" systemId="94ce86fb-07ff-4f43-adb8-93d2fa968ca2">
+            <cpix:URIExtXKey><FairPlay_URIExtXKey</cpix:URIExtXKey>
+            <cpix:HLSSignalingData playlist="master"><master_playlist_value></cpix:HLSSignalingData>
+            <cpix:HLSSignalingData playlist="media"><media_playlist_value></cpix:HLSSignalingData>
+        </cpix:DRMSystem>
+    </cpix:DRMSystemList>
+    <cpix:ContentKeyUsageRuleList></cpix:ContentKeyUsageRuleList>
+</cpix:CPIX>
+```
+
+#### CPIX keys as JSON
+
+Below is an example of a CPIX keys as JSON document with all `DRM Providers` selected. 
+
+```
+{
+    "key_id_hex": <key_id_hex_value>,
+    "content_key_hex": <content_key_hex_value>,
+    "iv_hex": <iv_hex_value>,
+    "playready_pssh_data": <playready_pssh_data_value>,
+    "playready_system_id": "9a04f079-9840-4286-ab92-e65be0885f95",
+    "widevine_drm_specific_data": <widevine_drm_specific_data_value>,
+    "widevine_drm_specific_data_with_key_id": <widevine_drm_specific_data_with_key_id_value>,
+    "widevine_system_id": "edef8ba9-79d6-4ace-a3c8-27dcd51d21ed",
+    "fairplay_system_id": "94ce86fb-07ff-4f43-adb8-93d2fa968ca2",
+    "fairplay_laurl": "skd://fairplay-license.vudrm.tech/v2/license/<content_id>"
+}
+```
+
+## Packaging Content with Widevine and Playready DRM
+
+Now we have retrieved the relevant documentation, we can now package content.
```

Comment:
Could you remove the second 'now' from the sentence, so it reads better?

Reply:
Will do!

Reply:
Actioned.